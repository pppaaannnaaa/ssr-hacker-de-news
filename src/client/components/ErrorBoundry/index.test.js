import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import ErrorBoundary from './index';

describe('<ErrorBoundary />', () => {
  let props;

  beforeEach(() => {
    props = {};
  });

  afterEach(cleanup);

  test('Renders <ErrorBoundary /> component correctly', () => {
    const { container, getByText } = render(<ErrorBoundary>Children</ErrorBoundary>);
    expect(container).toMatchSnapshot();
    expect(getByText('Children')).toBeInTheDocument();
  });
});
