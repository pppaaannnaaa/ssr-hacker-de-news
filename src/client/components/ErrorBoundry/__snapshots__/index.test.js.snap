// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorBoundary /> Renders <ErrorBoundary /> component correctly 1`] = `
<div>
  Children
</div>
`;
